tests/sdks/python/infrastructure/aws/test_aws_ssm_secret_provider.py

```
@@ -0,0 +1,76 @@
+
+        # Assert
+        assert actual is None
+        ssm_client.get_parameter.assert_called_once()
+
+    def Should_RaiseValueError_When_NameIsEmpty(self) -> None:
+        # Arrange
+        ssm_client = Mock()
+        sut = AwsSsmSecretProvider(ssm_client)
+
+        # Act
+        action = lambda: sut.get_secret("")
```

Ruff (0.15.9)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `action` as a `def`

(E731)